examples/callback_layer.html

```diff
@@ -12,13 +12,13 @@
         margin: 0;
       }
     </style>
-    <link rel="stylesheet" href="http://libs.cartocdn.com/cartodb.js/v3/3.15/themes/css/cartodb.css" />
+    <link rel="stylesheet" href="https://libs.cartocdn.com/cartodb.js/v3/3.15/themes/css/cartodb.css" />
   </head>
   <body>
     <div id="map"></div>
-    
+
     <!-- include cartodb.js library -->
-    <script src="http://libs.cartocdn.com/cartodb.js/v3/3.15/cartodb.js"></script>
+    <script src="https://libs.cartocdn.com/cartodb.js/v3/3.15/cartodb.js"></script>
     <script src="http://d3js.org/queue.v1.min.js"></script>
```

Comment:
https://d3js.org/queue.v1.min.js

Reply:
Oops! I think that example is not shown in the documentation but it's better to retrieve it with HTTPS as well :)